# NASA GeneLab Data Visualisation ecosystem

A collaboration between NASA GeneLab and the Gilroy Lab has led to a multi-omics, cross-species analysis of GeneLab omics data, creating a [new NASA Space Biology sponsored investigation](https://science.nasa.gov/science-news/bps/SpaceBiologyProgramNRAAwardsSep2020?fbclid=IwAR0QHO6hmNjyYWE-hRTL5eVB1mz0UyM-NgAwfFgCqDdZ3xs6RXVZO9oknBY) into the effects of spaceflight and microgravity simulations on plant-microbe interactions.

A suite of visualization tools to explore relationships between studies within the GeneLab data repository has been created by Richard Barker, a member of the Gilroy Lab at the University of Wisconsin, Madison, and co-lead of the GeneLab Plant Analysis Working Group ([AWG](https://genelab.nasa.gov/awg/charter)). These apps aggregate diverse data types and allow users to find associations between the distinct GeneLab accessions. This has allowed the identification of new correlations in the data leading to the generation of novel hypotheses and the creation of protocols for experimental validation.

#### &#x20;    [GeneLab X-Species Metadata ](https://astrobiology.botany.wisc.edu/astrobotany-toast/x-species-meta-analysis)– API-linked, GeneLab meta-data visualization that allows for the identification of links between studies with related meta-data fields.

#### &#x20;  [TOAST](https://astrobiology.botany.wisc.edu/astrobotany-toast/tutorial-metadata) - The AstroBotany multi-plant species and multi-omics explorer featuring microarray, RNAseq, epigenetics and proteomic data.&#x20;

#### &#x20;    [MANGO ISS MicroBiome](https://astrobiology.botany.wisc.edu/astromicrobiology-mango) - Explorer for the ISS microbiome and Microbial Observatory data: indoor microbiome study of the ISS surfaces and the ISS dust particle microbiome.&#x20;

#### &#x20;    [Rad-Bio App](https://astrobiology.botany.wisc.edu/astro-rad-bio-app) - Explorer for space radiation exposure experienced by each GeneLab experiment.&#x20;

***

The [Plant Reactome](https://plantreactome.gramene.org/index.php?option=com\_content\&view=featured\&Itemid=101\&lang=en) is an all-inclusive resource of plant pathways for basic research, genome analysis, pathway modeling, systems biology, and education. This is an open-source, open access, manually curated and peer-reviewed pathway database with the goals of conveying the rich information in the visual representations of plant pathways in a detailed, computationally accessible format. Pathway annotations are authored by expert biologists, in collaboration with the Reactome editorial staff and cross-referenced to many bioinformatics databases. These include project databases like Gramene, Ensembl, UniProt, ChEBI small molecule databases, PubMed, and Gene Ontology. The Plant Reactome resource provides an intuitive website to navigate pathway knowledge and a suite of data analysis tools to support the pathway-based analysis of complex experimental and computational data sets. So, come and check out this new resource!

#### &#x20;    [Pathway Browser ](https://plantreactome.gramene.org/PathwayBrowser/)– Visualize and interact with Plant Reactome biological pathways

#### &#x20;    [Analysis Tools ](https://plantreactome.gramene.org/PathwayBrowser/#TOOL=AT) – Species comparison, OMICS data analyses, pathway mapping, gene interaction

<figure><img src=".gitbook/assets/GeneLab_Humans_on_mars.jpeg" alt="" width="375"><figcaption><p>NASA Mars Inspiration Filter</p></figcaption></figure>

